### PR TITLE
Small update to the UI's look and feel

### PR DESCRIPTION
**Pull request by rafaelconde**
**- Summary**
This PR offers a very minor, pure visual update to the UI.
Take it as _quick and dirty_ fixes to how it looks, and hopefully the plan is to fan this spark into a flame, and consider a more thoughtful effort to rethink the User Interface.

**- Test plan**

demo-2

**- Description for the changelog**

Updates typography, colors, and some UI visual tweaks

**- A picture of a cute animal (not mandatory but encouraged)**

sloth

### CODE REVIEW

**Comment by erquhart** [comment]
@rafaelconde so fresh and so clean! Really happy to see this PR!

Things I love:

* That typography!
* Depth and motion
* Improved consistency
* No longer looks bad

I really only have one high level concern: contrast. We need it. Your mocks in #179 use gray for the header and sidebar, what do you think of pulling that into this change? Same for the form fields - at least for this iteration, we should have distinct borders for them. I think we can pull it off without hurting the look.

I mean *you*, *you* can pull it off. lol.

File: src/components/AppHeader/AppHeader.css [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 .appBar {
   background-color: var(--backgroundAltColor);
+  height: auto;
+  padding: 8px 2.4rem;
```
Comment (erquhart) [inline src/components/AppHeader/AppHeader.css]: I'd like to see us stick to pixels for units, but open to discussion on it if you feel strongly.

File: src/components/EntryListing/EntryListing.js [hunk 2]
```diff
@@ -56,7 +56,6 @@ export default class EntryListing extends React.Component {
           <p>{entry.getIn(['data', inferedFields.descriptionField])}</p>
           : inferedFields.remainingFields && inferedFields.remainingFields.map(f => (
             <p key={f.get('name')} className={styles.cardList}>
-              <span className={styles.cardListLabel}>{f.get('label')}:</span>{' '}
```
Comment (erquhart) [inline src/components/EntryListing/EntryListing.js]: I agree that this is redundant when viewing all entries of a given type, but it's necessary for the editorial workflow and search, where multiple entry types are present at once. If we're removing this we should add it back in for those views.

File: src/components/UI/card/Card.css [hunk 3]
```diff
@@ -3,31 +3,35 @@
 .card {
   composes: base container rounded depth;
   overflow: hidden;
-}
-
-.card > *:not(iframe, video, img, header, footer) {
```
Comment (erquhart) [inline src/components/UI/card/Card.css]: At a glance I'm uncertain why these ungainly selectors were added - are you sure we don't need them?

File: src/components/UI/card/Card.css [hunk 4]
```diff
@@ -3,31 +3,35 @@
...
+  transition: all .1s ease-in-out;
+  transform: translateY(0);
+  padding: 16px 24px;
 }
 
 .card > iframe,
 .card > video,
 .card > img {
-  max-width: 100%;
+  margin: -16px -24px 16px -24px;
```
Comment (erquhart) [inline src/components/UI/card/Card.css]: Nitpick: you can drop the fourth value.

File: src/components/AppHeader/AppHeader.css (hunk 1 above)
Comment (verythorough) [inline src/components/AppHeader/AppHeader.css]: I'd love to see us move away from pixels across the board (better for accessibility), but that's a much longer discussion, and one I'll probably lose. :)

**Comment by cassandraoid** [comment]
I can't wait for the facelift.

My notes:
- I agree we need more contrast. The cards with the same pattern as the background fades in a little bit too much for me. 
- The grid in the editor feels off to me. I like the contrast we have in it now, just wish it was a little prettier. (IE:Boxes feels shallow.)

**Comment by biilmann** [comment]
Comparing this:

screen shot 2017-05-01 at 11 42 37 am

With the current editing screen:

screen shot 2017-05-01 at 11 42 43 am

I think the current version is much easier to use. We need to make the UI look better, but it should never be at the cost of clarity and familiarity and I would bet good money that the current version will perform much, much better in any user testing with content editors that have never tried the CMS before.

I think the steps @erquhart has been taking in the current version to move away from the more discrete UI elements I had in my initial mockups, and towards something that looks way more like traditional form elements from a Wordpress admin UI or the like, is the right approach. The #1 goal should be to make it really, really easy to see what is the preview, what is a form element, where you can click, where the different fields are separated, etc.

File: src/components/UI/card/Card.css [hunk 5]
```diff
@@ -3,31 +3,35 @@
...
-  font-size: 18px;
+}
+
+.card p {
+  font-size: 14px;
+  font-weight: 400;
+  letter-spacing: 0;
+  line-height: 24px;
+  padding: 0;
+  color: #8c8c8c;
```
Comment (Benaiah) [inline src/components/UI/card/Card.css]: I don't think this should be hardcoded - we should add it as a text color in `theme.css`.

File: src/components/AppHeader/AppHeader.css (hunk 1 above)
Comment (imorente) [inline src/components/AppHeader/AppHeader.css]: I'm with @verythorough on using relative units instead of pixels. Any reason you prefer pixels, @erquhart?

Comment (Benaiah) [inline src/components/AppHeader/AppHeader.css]: @imorente the reasons for using `px` are consistency with existing CSS and the fact that `rem`s and `em`s can cause half-pixel boundaries, which are undesirable. I have mixed feelings - those are both significant issues, but the accessibility problem is a hard one to ignore. For now, I think we should stick with `px` for the simple reason that we shouldn't make a transition to relative units ad-hoc, even if we do end up using them at some point in the future.

Comment (erquhart) [inline src/components/AppHeader/AppHeader.css]: Yep, the idea is:
1. Use pixels everywhere for now - simple and declarative.
2. When we get an actual design system in place and get rid of all the hodge podge, bring in the ems and rems. It's more complex, but the benefits are there if we implement properly.

File: src/components/EntryListing/EntryListing.js (hunk 2 above)
Comment (rafaelconde) [inline src/components/EntryListing/EntryListing.js]: I'll leave this to you, I'll revert it back 👍

**Comment by erquhart** [comment]
Rebased and added a commit to address the following:

* Improved entry listing contrast by replacing shadow/border with input field border styling
* Removed entry listing flex treatment - stretching cards can lead to wonky image presentation and degrade the overall appearance
* Removed leftover toolbar bottom border
* Slight code improvements

**Comment by biilmann** [comment]
There's still some contract issues for some widgets - the image and date widget for examlple:

screen shot 2017-05-03 at 2 17 39 pm

And the bottom bar also needs some clearer definition imo - right now it floats together with the forms panel.

**Comment by erquhart** [comment]
@biilmann I was thinking we could settle a bit on this PR, and fix the remaining contrast issues in a subsequent PR. Widgets and the editor footer are both on my radar.

**Comment by calavera** [comment]
The contrast of the image widget is still a little bit off, it's very easy to miss. Can we make the border at least slightly more visible? Why are we using such a light grey 
screenshot from 2017-05-05 10-58-14
in every border?

**Comment by erquhart** [comment]
@rafaelconde if you can improve the contrast issues that David and Matt pointed out, we should be good to merge.

**Comment by rafaelconde** [comment]
I addressed the contrast of popover and borders.

* I don't believe that removing the drop-shadow on the cards is the best way to address the contrast issue (now some cards have depth, some don't 🙃).
* I also don't think that removing the adaptive grid on the card layout is the best way to address the super wide orphan elements — this way we effectively will never have the grid taking advantage of the available space, it looks odd IMO.

I also think that this could and should be addressed on a separate PR 🙏

**Comment by erquhart** [comment]
@rafaelconde thanks, and yeah there's plenty to improve, and there will be more opportunity to do so. Just want to get this thing merged right now!

@biilmann @calavera if there's no feedback from either of you by EOD, I'm planning to merge.

**Comment by calavera** [comment]
My main issue is that I can barely see the border in the image widget. That has not been addressed.

**Comment by rafaelconde** [comment]
screenshot 2017-05-10 14 01 12

Assuming @calavera is talking about this — I literally don't know how to change the border on that.

**Comment by imorente** [comment]
@rafaelconde, I found the styles for the image widget 🎉 

They're here: https://github.com/netlify/netlify-cms/blob/visual-tweaks/src/components/Widgets/ImageControl.js#L113

**Comment by rafaelconde** [comment]
Thanks @imorente !

**Comment by erquhart** [comment]
screen shot 2017-05-12 at 11 59 38 am

@calavera good?

**Comment by calavera** [comment]
> @calavera good?

yes

**Pull request merged**